docs/widgets_adwaita.md

```diff
@@ -484,6 +484,42 @@ renderable AboutWindow
 - `license: string`
 
 
+## ToastOverlay
+
+```nim
+renderable ToastOverlay of BaseWidget
+```
+
+An overlay to display Toast messages that can be dismissed manually and automatically!<br>
+Use `newToast` to create an `Toast`.
+`Toast` has the following properties that can be assigned to:
+- actionName
```

Comment:
On the one hand, we currently don't support that system (can we even at all? I need to jot that down in my notes if we ever will or won't). On the other it's an available property.

Remove? Keep in there?

Reply:
I think its fine to leave out for now, we can still add it should it become necessary.

Reply:
Alrighty, removed in that case.